packages/docs/scripts/bump-docs.js

```diff
@@ -65,6 +66,27 @@ function genCliDocs (packagesDir) {
   return { 'CLI REFERENCE': commands }
 }
 
+function genVouchDocs(packagesDir) {
+  const docsDir = path.resolve(packagesDir, 'docs', 'docs')
+  const builtDocs = path.resolve(docsDir, 'docs')
+  const vouchDir = path.resolve(packagesDir, 'vouching')
+  const vouchContractsDir = path.resolve(vouchDir, 'contracts')
+  const libDir = path.resolve(vouchDir, 'node_modules', 'zos-lib')
+  const tplDir = path.resolve(vouchDir, 'node_modules', 'tpl-contracts-zos')
```

Comment:
Shouldn't we change this to `tpl-contracts-eth`?
```suggestion
  const tplDir = path.resolve(vouchDir, 'node_modules', 'tpl-contracts-eth')
```

Reply:
This should go away upon rebasing.